README.md

```diff
@@ -8,25 +8,14 @@ In production, the Docker image is mounted into container running Keycloak. See
 
 ## Configuration
 
-The theme can be configured to prompt a user to accept the **terms of service** on their first login. This is off by default, but to enable it, modify the following section in `renku-realm.json`:
+Keycloak can be configured to prompt a user to accept the **terms of service** on their first login. In Keycloack version 20, this can be done in the admin UI under the `Authentication` section for the realm.
 
-```
-{
-    "alias": "terms_and_conditions",
-    "name": "Terms and Conditions",
-    "providerId": "terms_and_conditions",
-    "enabled": false,
-    "defaultAction": true,
-    "priority": 20,
-    "config": {}
-}
-```
+Select the `Required Actions` tab, make sure `Terms and Conditions` is enabled and set it as default action as well.
 
-and set `enabled` to `true`.
+More details are available in this [stack overflow post](
+https://stackoverflow.com/a/77638989/5804638).
```

Comment:
I don't think that this post is relevant. Flipping the two bits in the Keycloak console enables the required action during registration. You can see it in action here: https://ci-renku-3480.dev.renku.ch.